tket/src/serialize/pytket/encoder.rs

```diff
@@ -56,6 +58,11 @@ pub struct PytketEncoderContext<H: HugrView> {
     unsupported: UnsupportedTracker<H::Node>,
     /// A registry of already-encoded opaque subgraphs.
     opaque_subgraphs: OpaqueSubgraphs<H::Node>,
+    /// Subgraphs in `opaque_subgraph` that could not be emitted as an opaque
```

Comment:
```suggestion
    /// Subgraphs in `opaque_subgraphs` that could not be emitted a opaque barriers
```
Unless we can make this just one - you'd still need to add the Option here tho